src/hound/git/git.go

```diff
@@ -42,7 +36,11 @@ func HeadHash(dir string) (string, error) {
 }
 
 func Pull(dir string) (string, error) {
-	if err := pull(dir); err != nil {
+	cmd := exec.Command("git", "pull")
+	cmd.Dir = dir
+	out, err := cmd.CombinedOutput()
```

Comment:
Instead of combining the output, why don't we just give stdout and stderr of the parent process to the child. `cmd.Stdout = os.Stdout` and `cmd.Stderr = os.Stderr`. That way the streams will be intact.